packages/interwovenkit-react/src/pages/wallet/tabs/portfolio/VaultSection.module.css

```
@@ -0,0 +1,226 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+
+  padding: 12px 16px;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)